### PR TITLE
[Issue 4047] Catch up docs for functions/sink/source in Admin CLI

### LINKED ISSUE #4047: Need to catch up docs for functions/sink/source in Admin CLI
**Describe the bug**
Some of the subcommands of functions/sink/source are not documented in [Pulsar admin CLI](http://pulsar.apache.org/docs/en/pulsar-admin/), such as `sink/source status`. We need update the doc to catch up the work.

### PR DESCRIPTION
**Pull request by murong00**
### Motivation

Fixes #4047

### Modifications

Add the missing doc for functions/sink/source and some minor updates.

### CODE REVIEW

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java [hunk 1]
```diff
@@ -280,9 +280,9 @@ void processArguments() throws Exception {
         @Parameter(names = "--sliding-interval-duration-ms", description = "The time duration after which the window slides")
         protected Long slidingIntervalDurationMs;
         // for backwards compatibility purposes
-        @Parameter(names = "--autoAck", description = "Whether or not the framework will automatically acknowleges messages", hidden = true)
+        @Parameter(names = "--autoAck", description = "Whether or not the framework will automatically acknowledges messages", hidden = true)
```
Comment (jiazhai) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java]: should be acknowledge withtout "s"

**Comment by jiazhai** [comment]
@murong00 Thanks for the work, left a minor comments

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java (hunk 1 above)
Comment (murong00) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java]: Changed!

**Comment by sijie** [comment]
run cpp tests
run java8 tests

**Pull request merged**